### PR TITLE
added some file extensions

### PR DESCRIPTION
**Pull request by Iftakharpy**
added some file extensions for videos and sorted the list

### CODE REVIEW

**Comment by sindresorhus** [comment]
> sorted the list

That should be a separate follow-up PR.

File: video-extensions.json [hunk 1]
```diff
@@ -1,35 +1,57 @@
...
+	'ogv',
+	'ogx',
+	'qt',
+	'rm',
+	'rmvb',
+	'roq',
+	'svi',
+	'ts',
+	'tsa',
+	'tsv',
```
Comment (sindresorhus) [inline video-extensions.json]: This is not valid JSON.

**Review by Iftakharpy — COMMENTED** [review]
fixed the file format for json

File: video-extensions.json [hunk 2]
```diff
@@ -21,15 +35,23 @@
...
 	"rm",
 	"rmvb",
 	"roq",
 	"svi",
+	"ts",
+	"tsa",
+	"tsv",
 	"vob",
+	"wav",
+	"wave",
```
Comment (sindresorhus) [inline video-extensions.json:52]: These are sound, not video.

**Comment by sindresorhus** [comment]
Many of the extensions are incorrect, so I must ask you to go through the list again, and also provide a source for each added extension as proof.

**Pull request closed without merging**